src/domain/hanke/edit/HankeForm.test.tsx

```diff
@@ -60,24 +61,25 @@ async function uploadAttachment({
 
 function initFileGetResponse(response: HankeAttachmentMetadata[]) {
   server.use(
-    rest.get('/api/hankkeet/:hankeTunnus/liitteet', async (req, res, ctx) => {
-      return res(ctx.status(200), ctx.json(response));
+    http.get('/api/hankkeet/:hankeTunnus/liitteet', async () => {
+      return HttpResponse.json(response, { status: 200 });
     }),
   );
 }
 
 function initFileUploadResponse() {
   server.use(
-    rest.post('/api/hankkeet/:hankeTunnus/liitteet', async (req, res, ctx) => {
-      return res(ctx.delay(), ctx.status(200));
+    http.post('/api/hankkeet/:hankeTunnus/liitteet', async () => {
+      await delay();
+      return new HttpResponse(null, { status: 200 });
```

Comment:
This can also be written as `return HttpResponse.json(null);`

Reply:
That sets the `Content-Type: "application/json"` header, which we might not want with an empty response.

Reply:
True. In real-life this POST would not return nothing but a JSON response like:
```
{
    "id": "bdb862ee-4592-4073-89ff-5c631b9cb418",
    "fileName": "paatos.pdf",
    "contentType": "application/pdf",
    "size": 35764,
    "createdByUserId": "5f893af3-f7c0-433b-bf50-08f88fbc43a7",
    "createdAt": "2024-09-06T09:11:28.601676383Z",
    "hankeTunnus": "HAI24-10"
}
```
The returned data is not checked in these tests though.